samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/ASBS_1/AsbEndpoint/Program.cs

```diff
@@ -13,37 +13,20 @@ static async Task Main()
         Console.Title = "Samples.Azure.ServiceBus.AsbEndpoint";
         var endpointConfiguration = new EndpointConfiguration("Samples.Azure.ServiceBus.AsbEndpoint");
         endpointConfiguration.EnableInstallers();
-        endpointConfiguration.UsePersistence<InMemoryPersistence>();
+        endpointConfiguration.UsePersistence<NonDurablePersistence>();
         endpointConfiguration.UseSerialization<NewtonsoftSerializer>();
         endpointConfiguration.AddDeserializer<XmlSerializer>();
 
-        var transport = endpointConfiguration.UseTransport<AzureServiceBusTransport>();
-        var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus.ConnectionString");
+        var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus_ConnectionString");
         if (string.IsNullOrWhiteSpace(connectionString))
         {
             throw new Exception("Could not read the 'AzureServiceBus.ConnectionString' environment variable. Check the sample prerequisites.");
         }
-        transport.ConnectionString(connectionString);
-
-        #region connect-asb-side-of-bridge
```

Comment:
https://github.com/Particular/docs.particular.net/blob/master/samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/sample.md needs to be updated to reflect the changes made